### DISCUSSION TITLE
中文生僻字显示不了

### DISCUSSION THREAD

**Discussion by Cyan99** [note286/xduts#88] [Q&A] [answered]
引用的参考文献里有个作者名字包含”烺“，编译后显示方框F，这个应该怎么处理呀

**Comment by note286** [comment, marked as answer]
根据手册切换中文字体配置。

**Reply by zhongdiandaoda** [reply]
> 引用的参考文献里有个作者名字包含”烺“，编译后显示方框F，这个应该怎么处理呀

咋解决的啊，我是fandol字体，要换别的吗

**Reply by note286** [reply]
是的，换成其他的都可以解决。

**Reply by note286** [reply]
https://github.com/note286/xduts/discussions/16

**Reply by Cyan99** [reply]
已更改，感谢
